### PR TITLE
fix(demo) fix every broken links

### PR DESCRIPTION
**Pull request by oussamahamdaoui**
### Reference to the issue

`https://github.com/AxaGuilDEv/react-toolkit/issues/46`

### CODE REVIEW

File: examples/demo/public/environment.dev.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "apiBaseUrl": "http://localhost:5000/api/{path}"
+  "apiBaseUrl": "https://api.mlab.com/api/1/databases/toolkitaxa/collections/{path}",
```
Comment (guillaumechervetaxa) [inline examples/demo/public/environment.dev.json]: yep oussama,
 in dev mode:
"apiBaseUrl": "http://localhost:5000/api/{path}"

is the good one ^^

Very nice PR !

File: examples/demo/public/environment.json [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {
-  "apiUrl": "https://api.mlab.com/api/1/databases/toolkitaxa/collections/{path}"
+  "apiUrl": "https://api.mlab.com/api/1/databases/toolkitaxa/collections/{path}",
+  "baseUrl" : "build/"
```
Comment (guillaumechervetaxa) [inline examples/demo/public/environment.json]: have you think to update the release srcipts?

Comment (oussamahamdaoui) [reply examples/demo/public/environment.json]: i was going to make another pull request for this

File: examples/demo/public/environment.dev.json [hunk 3]
```diff
@@ -1,3 +1,4 @@
 {
-  "apiBaseUrl": "http://localhost:5000/api/{path}"
+  "apiBaseUrl": "http://localhost:5000/api/{path}",
```
Comment (guillaumechervetaxa) [inline examples/demo/public/environment.dev.json]: Du coup il faut renomer ici aussi en "apiUrl" (enfin je crois)

**Pull request merged**